### PR TITLE
typo fix in log messages

### PR DESCRIPTION
**Pull request by rylorin**
Some minor corrections in log messages related to connection

### CODE REVIEW

File: src/core/io/socket.ts [hunk 1]
```diff
@@ -159,6 +159,7 @@ export class Socket {
     // disconnect TCP socket.
 
     this.client?.end();
+    // delete this.client;
```
Comment (Tsopic) [inline src/core/io/socket.ts:162]: I assume this change is not needed

File: src/core/io/socket.ts [hunk 2]
```diff
@@ -411,7 +412,7 @@ export class Socket {
       this.controller.emitEvent(EventName.disconnected);
     }
 
-    this.controller.resume();
+    this.controller.pause();
```
Comment (Tsopic) [inline src/core/io/socket.ts:415]: It's a big logical difference, care to explain?

File: src/core/io/socket.ts (hunk 1 above)
Comment (rylorin) [reply src/core/io/socket.ts:162]: you are right.

File: src/core/io/socket.ts (hunk 2 above)
Comment (rylorin) [reply src/core/io/socket.ts:415]: I'm not sure it's a big difference as starting to send messages when the socket is closed won't send anything, but the correct behaviour is to pause the communication.

**Review by Tsopic — APPROVED** [review]

**Pull request merged**